docs/mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: Protect AI Docs
+site_name: NB Defense: Secure Jupyter Notebooks
```

Comment:
You'll have to wrap this in `""` to escape the `:`

Reply:
Thanks. New commit should fix it.